docs/config.rst

```diff
@@ -90,7 +90,7 @@ Those configuration options are documented below:
     tags. Not setting this value is equivalent to a catch-all and will not
     filter out any values.
 
-    Does not affect ``captureMessage`` or when non-error object is passed in
+    Does not affect when non-error object is passed in
```

Comment:
This won't be true anymore with this PR, as `Raven.captureException({foo: 'bar'})`, will end up calling `Raven.captureMessage` https://github.com/getsentry/raven-js/blob/master/src/raven.js#L378-L389